waspstation/code/game/turf/open/floor/whitesands.dm

```diff
@@ -7,11 +7,12 @@
 	environment_type = WHITESANDS_SAND_ENV
 	initial_gas_mix = OPENTURF_LOW_PRESSURE //Fallback, and used to tell the AACs that this is the exterior
 	digResult = /obj/item/stack/ore/glass
-	var/static/datum/gas_mixture/immutable/whitesands_planet/whitesands_gas = new
 
 /turf/open/floor/plating/asteroid/whitesands/Initialize()
 	..()
-	air = whitesands_gas
+	if (GLOB.ws_planet_atmos.ws_moles_amount == null)
```

Comment:
I would love a recommendation as to how to better handle this.

Reply:
there probably is, but I can't think of anything at the moment